### ISSUE TITLE
Error building Player: CommandInvokationFailure: Failed to re-package resources. See the Console for details.

### ISSUE THREAD

**Issue by galaxyly** [playgameservices/play-games-plugin-for-unity#1050]
when i export a google project get this error:

Error building Player: CommandInvokationFailure: Failed to re-package resources. See the Console for details.
/Users/liuchris/Library/Android/sdk/build-tools/23.0.2/aapt package --auto-add-overlay -v -f -m -J gen -M AndroidManifest.xml -S "res" -I "/Users/liuchris/Library/Android/sdk/platforms/android-23/android.jar" -F bin/resources.ap_ --extra-packages com.google.example.games.mainlibproj

stderr[
AndroidManifest.xml:17: error: Error: No resource found that matches the given name (at 'value' with value '@integer/google_play_services_version').

<img width="598" alt="2016-03-15 12 20 02" src="https://cloud.githubusercontent.com/assets/17842118/13769112/e8ee8172-eab7-11e5-868f-fa7251381b05.png">

==============================================================================
if i copy the google-play-services_lib to my  Assets/Plugins/Android/  , yes , it can work once , but when export project over your the google-play-services_lib all file has gone (?????) and you will get other erro

![qq20160315-0 2x](https://cloud.githubusercontent.com/assets/17842118/13769566/8641aa6e-eabb-11e5-8304-b2a224e4d924.png)

**Comment by bar2016** [comment]
Well , if i understood the developer of this plugin, i think that he said , when you build the apk
you don't want it to gain any more weight, so its simply removes it, yea thats annoying every time i build up i need to add the folder again, i found a simple way , compress it to a rar drop it in the plugins/android/ and then every time you need to build you just extract it.

**Comment by skwkddkemf** [comment]
Unity in after eclipse export in build, Do you put up for Android device by? That is, gpgs active You don't occur to lead this error?

**Comment by galaxyly** [comment]
i use Android studio (not eclipse) , when i import the model to the project , i found lot's of error need fix , dependence 。。。so i give up this plugins :(

**Comment by skwkddkemf** [comment]
yes...i think so too... Are you proceed with the development in native code ...?

**Comment by galaxyly** [comment]
yes , i only need googlePlayService account login

**Comment by mliitfall** [comment]
i have this issue too, thought not imported as google project, just build it to apk. So how to fix this?

**Comment by claywilkinson** [comment]
@galaxyly - did you get this resolved?  All the other people who are having problems, I'd suggest making a separate issue with your specific errors - it looks like there are at least 2 conversations happing here :/
